elmslie-plugin/src/main/resources/templates/PresentationModels.kt.ftl

```diff
@@ -0,0 +1,39 @@
+package ${presentation_package_name}
+
+import vivid.money.elmslie.core.store.Store
+
+internal typealias ${domain_name}Store =
+        Store<${domain_name}Event, ${domain_name}Effect, ${domain_name}State>
+
+internal data class ${domain_name}State(
+    val isRefreshing: Boolean = false,
```

Comment:
Shouldn't this be at least optinal? This not neceaary for every screen and definately redundant for pure kotlin modules